### DISCUSSION TITLE
How to show Stream WebRTCApp with Publish URL in Android Studio using Java

### DISCUSSION THREAD

**Discussion by yudentaa521** [ant-media/Ant-Media-Server#5650] [Q&A]
Hello..

I want to show my video stream WebRTCApp with Publish URL but error :
"Uncaught SyntaxError: Unexpected token ="

This is my coding:

webView = findViewById(R.id.webView);
        WebSettings webSettings = webView.getSettings();
        webSettings.setJavaScriptEnabled(true); // Aktifkan JavaScript jika diperlukan

        webView.setWebViewClient(new WebViewClient()); // Menggunakan WebViewClient bawaan

        String url = "http://my.ip/WebRTCApp/play.html?id=SFTBwIVm4pL98RPo1690928901152";

        // Memecah URL
        String[] parts = url.split("\\?");
        String baseUrl = parts[0];
        System.out.println("Parts apa hayo " + baseUrl);
        String query = parts[1];

        // Menggabungkan URL kembali
        String newUrl = baseUrl + "?" + query;
        // Muat halaman web yang diinginkan
        webView.loadUrl(newUrl);

**Comment by Mohit-3196** [comment]
Hi @yudentaa521,
How are you?

I haven't tried webView so if anyone from the community has some experience with it, I would welcome them to contribute to this.
Before that, I have some questions for you in order to understand the use case and related things.
I see you are using the WebRTCApp application which is an application in the community edition of the Ant Media Server and the community edition doesn't support WebRTC playback.
In case you are trying to play the HLS stream using the Android SDK, the Android SDK is the WebRTC android SDK so it doesn't support HLS.

Also, I would like to understand more about your use case and what are the things that you are looking to achieve.

Thank you
